### ISSUE TITLE
"Missing React element for debugID" warnings when rendering BarChart

### ISSUE THREAD

**Issue by dorsha** [recharts/recharts#139]
Got the following warnings when rendering the below code:
![image](https://cloud.githubusercontent.com/assets/5429033/16584273/a11966be-42c3-11e6-806d-9b2d5db0a51f.png)

```
import React, { PropTypes } from 'react';
import { BarChart, Bar, XAxis, YAxis, Tooltip, Legend, CartesianGrid } from 'recharts';

const SectionBarChart = ({ data, style }) =>
  <div className="section-bar-chart" style={style}>
    <BarChart width={800} height={400} data={data}>
      <XAxis dataKey="name" />
      <YAxis />
      <CartesianGrid strokeDasharray="3 3" />
      <Tooltip />
      <Legend />
      <Bar dataKey="pv" fill="#8884d8" />
      <Bar dataKey="uv" fill="#82ca9d" />
    </BarChart>
  </div>
;
SectionBarChart.propTypes = {
  data: PropTypes.oneOfType([
    PropTypes.object,
    PropTypes.array
  ]),
  style: PropTypes.object
};

export default SectionBarChart;
```

Data:
```
const data = [
    { "name": "Page A", "uv": 4000, "pv": 2400, "amt": 2400 },
    { "name": "Page B", "uv": 3000, "pv": 1398, "amt": 2210 },
    { "name": "Page C", "uv": 2000, "pv": 9800, "amt": 2290 },
    { "name": "Page D", "uv": 2780, "pv": 3908, "amt": 2000 },
    { "name": "Page E", "uv": 1890, "pv": 4800, "amt": 2181 },
    { "name": "Page F", "uv": 2390, "pv": 3800, "amt": 2500 },
    { "name": "Page G", "uv": 3490, "pv": 4300, "amt": 2100 }
]
```

React: 15.2.0
recharts: 0.12.3

Warnings does not appear when running in NODE_ENV=production, since React does not print them.

![image](https://cloud.githubusercontent.com/assets/5429033/16584304/eb2a2eaa-42c3-11e6-960e-912f367f00ce.png)

**Comment by xile611** [comment]
It seems the bug of latest [React](https://github.com/facebook/react/issues/7187);

**Comment by dorsha** [comment]
I see, do you want me to close the issue?

**Comment by xile611** [comment]
@dorsha You can keep this issue open until React fix this bug.

**Comment by dorsha** [comment]
Fixed in React 15.2.1 - Verified.
